clients/admin-ui/src/features/config-wizard/ReviewSystemForm.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  Divider,
+  FormLabel,
+  Heading,
+  HStack,
+  Stack,
+  Tag,
+  Text,
+} from "@fidesui/react";
+import { Form, Formik } from "formik";
+import type { NextPage } from "next";
+import React from "react";
+
+import { useGetSystemByFidesKeyQuery } from "../system/system.slice";
+import { useGetOrganizationByFidesKeyQuery } from "./organization.slice";
+
+const ReviewSystemForm: NextPage<{
+  handleChangeStep: Function;
+  handleCancelSetup: Function;
+}> = ({ handleCancelSetup, handleChangeStep }) => {
+  // TODO FUTURE: Need a way to check for an existing fides key from the start of the wizard
+  // not just use this default
+  const { data: existingSystem } = useGetSystemByFidesKeyQuery(
+    "default_organization"
+  );
+  const { data: existingOrg } = useGetOrganizationByFidesKeyQuery(
+    "default_organization"
+  );
+
+  const initialValues = {
+    name: existingOrg?.name ?? "",
+    system_name: existingSystem?.name ?? "",
+    // TODO FUTURE:
+    // system_key: existingSystem?.key,
```

Comment:
what is this key?

Reply:
@earmenda I'm waiting to hear back from Simon on this. It seems like this will be something that is generated for the user, but I don't see anything referencing a key in the API, however system key exists in the designs. For now, I am leaving the code commented out with a future todo so there is no risk in shipping.